modules/ebi-metagenomics/pyrodigal/meta.yml

```diff
@@ -1,79 +1,83 @@
-      homepage: https://github.com/hyattpd/Prodigal
-      documentation: https://github.com/hyattpd/prodigal/wiki
-      tool_dev_url: https://github.com/hyattpd/Prodigal
-      doi: "10.1186/1471-2105-11-119"
+  - "pyrodigal":
+      description: "Pyrodigal is a Python module that provides bindings to Prodigal
+        (ORF finder for microbial sequences) using Cython."
+      homepage: "https://pyrodigal.readthedocs.org/"
+      documentation: "https://pyrodigal.readthedocs.org/"
+      tool_dev_url: "https://github.com/althonos/pyrodigal/"
+      doi: "10.21105/joss.04296"
       licence: ["GPL v3"]
-      identifier: biotools:prodigal
+      identifier: ""
 input:
   - - meta:
         type: map
         description: |
           Groovy Map containing sample information
           e.g. [ id:'test', single_end:false ]
-    - genome:
+    - fasta:
         type: file
-        description: fasta/fasta.gz file
+        description: FASTA file
+        pattern: "*.{fasta.gz,fa.gz,fna.gz}"
   - - output_format:
         type: string
-        description: Output format ("gbk"/"gff"/"sqn"/"sco")
+        description: Output format
+        pattern: "{gbk,gff}"
 output:
-  - gene_annotations:
+  - annotations:
       - meta:
           type: map
           description: |
             Groovy Map containing sample information
             e.g. [ id:'test', single_end:false ]
-      - ${prefix}.${output_format}.gz:
+      - "*.${output_format}.gz":
           type: file
-          description: gene annotations in output_format given as input
-          pattern: "*.{output_format}"
-  - nucleotide_fasta:
+          description: Gene annotations. The file format is specified via input channel
+            "output_format".
+          pattern: "*.{gbk,gff}.gz"
+  - fna:
       - meta:
           type: map
           description: |
             Groovy Map containing sample information
             e.g. [ id:'test', single_end:false ]
-      - ${prefix}.fna.gz:
+      - "*.fna.gz":
           type: file
           description: nucleotide sequences file
-          pattern: "*.{fna}"
-  - amino_acid_fasta:
+          pattern: "*.{fna.gz}"
+  - faa:
       - meta:
           type: map
           description: |
             Groovy Map containing sample information
             e.g. [ id:'test', single_end:false ]
-      - ${prefix}.faa.gz:
+      - "*.faa.gz":
           type: file
           description: protein translations file
-          pattern: "*.{faa}"
-  - all_gene_annotations:
+          pattern: "*.{faa.gz}"
+  - score:
       - meta:
           type: map
           description: |
             Groovy Map containing sample information
             e.g. [ id:'test', single_end:false ]
-      - ${prefix}_all.txt.gz:
+      - "*.score.gz":
           type: file
-          description: complete starts file
-          pattern: "*.{_all.txt}"
+          description: all potential genes (with scores)
```

Comment:
maybe describe what kind of file a .score is? tab separated, comma separated, etc... not that important though since it's not even "our" module haha

Reply:
yeah, I have some notes on my own that I plan to use to submit a PR to nf-core... _soon_